lib/logstash/agent.rb

```diff
@@ -48,6 +48,10 @@ class LogStash::Agent < Clamp::Command
     I18n.t("logstash.agent.flag.configtest"),
     :attribute_name => :config_test
 
+  option ["--auto-download"], :flag,
```

Comment:
let's make this '--[no-]auto-download' so folks can say `--no-auto-download` if they wish